backend/canisters/community/impl/src/lib.rs

```diff
@@ -686,6 +691,38 @@ impl Data {
                 .push(user_id, CommunityCanisterEvent::Achievement(achievement));
         }
     }
+
+    pub fn get_bot_permissions(&self, bot_user_id: &UserId) -> Option<&SlashCommandPermissions> {
+        self.bot_permissions.get(bot_user_id)
+    }
+
+    pub fn get_user_permissions(&self, user_id: &UserId, channel_id: &ChannelId) -> Option<SlashCommandPermissions> {
```

Comment:
`get_user_permissions` returns `Option<SlashCommandPermissions>`, that doesn't seem right.

Maybe `SlashCommandPermissions` should be renamed to something which applies to users and bots.

Reply:
Ahh right, this is getting the user permissions specifically for executing bot commands.
So the return type is fine, but the method name should be clearer.
`get_user_permissions_for_bot_commands` or something.